assets/src/edit-story/components/header/titleMissingDialog.js

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { Plain } from '../button';
+import Dialog from '../dialog';
+import Link from '../link';
+
+const Paragraph = styled.p`
+  font-family: ${({ theme }) => theme.fonts.body1.family};
+  font-size: ${({ theme }) => theme.fonts.body1.size};
+  line-height: ${({ theme }) => theme.fonts.body1.lineHeight};
+  letter-spacing: ${({ theme }) => theme.fonts.body1.letterSpacing};
+`;
+
+const GUIDELINES =
+  'https://amp.dev/documentation/guides-and-tutorials/start/create_successful_stories/#title';
+
+function TitleMissingDialog({ open, onIgnore, onFix }) {
+  return (
+    <Dialog
+      open={open}
+      onClose={onFix}
```

Comment:
Should just closing the dialog maybe do nothing else? Currently, it focuses on the title even if the user doesn't choose the adding title option. WDYT?

Reply:
Fixed in a9022700c.